test/test_rng.cpp

```diff
@@ -354,4 +354,67 @@ TEST_F(RNGTest, RandLikeReduction) {
   testValidate(fec.fusion(), {out}, {t0}, {t3}, __LINE__, __FILE__);
 }
 
+//! This is the same as the Uniform test, but we compare against
+//! functional_uniform in which we provide a seed and offset.
+TEST_F(RNGTest, FunctionalUniform) {
```

Comment:
Could you also test the case where there is only functional uniform, no stochastic uniform in the fusion?

Reply:
This test now also tests without any stochastic ops present.